server/Login.class.php

```diff
@@ -685,4 +677,12 @@ private function processException($exception, &$response){
 		}
 		$response['success'] = false;
 	}
+
+	private function sendHtmlFormattedEmail($toEmail, $subject, $body, $fromEmail) {
```

Comment:
Is it really necessary to have a `fromEmail` parameter if this application will only ever use the noreply email? Even the `sendHtmlFormattedEmail` function defaults to the noreply.

I would be on-board to get rid of `fromEmail` entirely, but maybe there are use-cases for it I'm not aware of 🤷‍♂️

Reply:
It this class, probably not. Good idea, removed that parameter and made it a local variable to the function.